www/src/pages/ja/deployment/docker.md

```diff
@@ -0,0 +1,284 @@
+> - _Next.js は[一般的な使用方法に関する匿名のテレメトリーデータ](https://nextjs.org/telemetry)を収集します。ビルド時にテレメトリを無効にするには、`ENV NEXT_TELEMETRY_DISABLED 1` の最初のインスタンスをアンコメントします。実行時にテレメトリを無効にするには、2 番目のインスタンスをアンコメントしてください。_
+
+</div>
+</details>
+
+## Build and Run Image Locally
+
+Build and run this image locally with the following commands:
+
+```bash
+docker build -t ct3a-docker --build-arg NEXT_PUBLIC_CLIENTVAR=clientvar .
+docker run -p 3000:3000 -e DATABASE_URL="database_url_goes_here" ct3a-docker
+```
+
+Open [localhost:3000](http://localhost:3000/) to see your running application.
+
+## ローカルでイメージをビルドし、実行する
+
+このイメージを以下のコマンドでローカルにビルドして実行します：
+
+```bash
+docker build -t ct3a-docker --build-arg NEXT_PUBLIC_CLIENTVAR=clientvar .
+docker run -p 3000:3000 -e DATABASE_URL="database_url_goes_here" ct3a-docker
+```
+
+[localhost:3000](http://localhost:3000/)を開き、実行中のアプリケーションを確認します。
+
+## Docker Compose
+
+You can also use Docker Compose to build the image and run the container.
+
+また、Docker Compose を使用してイメージを構築し、コンテナを実行することもできます。
+
+<details>
+    <summary>
+      Follow steps 1-4 above, click here, and include contents in <code>docker-compose.yml</code>:
+    </summary>
+<div class="content">
+
+```yaml
+version: "3.9"
+services:
+  app:
+    platform: "linux/amd64"
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        NEXT_PUBLIC_CLIENTVAR: "clientvar"
+    working_dir: /app
+    ports:
+      - "3000:3000"
+    image: t3-app
+    environment:
+      - DATABASE_URL=database_url_goes_here
+```
+
+Run this using the `docker compose up` command:
+
+これを `docker compose up` コマンドで実行します：
+
+```bash
+docker compose up
+```
+
+Open [localhost:3000](http://localhost:3000/) to see your running application.
+
+[localhost:3000](http://localhost:3000/)を開き、実行中のアプリケーションを確認します。
+
+</div>
+</details>
+
+## Deploy to Railway
+
+You can use a PaaS such as [Railway's](https://railway.app) automated [Dockerfile deployments](https://docs.railway.app/deploy/dockerfiles) to deploy your app. If you have the [Railway CLI installed](https://docs.railway.app/develop/cli#install) you can deploy your app with the following commands:
+
+## Railway へのデプロイ
+
+アプリのデプロイには、[Railway](https://railway.app)のような自動化された[Dockerfile deployments](https://docs.railway.app/deploy/dockerfiles) などの PaaS を使用することができます。[ailway CLI をインストールしている場合](https://docs.railway.app/develop/cli#install)は、以下のコマンドでアプリをデプロイすることができます：
+
+```bash
+railway login
+railway init
+railway link
+railway up
+railway open
+```
+
+Go to "Variables" and include your `DATABASE_URL`. Then go to "Settings" and select "Generate Domain." To view a running example on Railway, visit [ct3a-docker.up.railway.app](https://ct3a-docker.up.railway.app/).
+
+変数」にて、`DATABASE_URL`を指定します。その後、"Settings" に行き、"Generate Domain" を選択します。Railway 上での実行例を見るには、[ct3a-docker.up.railway.app](https://ct3a-docker.up.railway.app/)にアクセスしてください。
```

Comment:
Is railway.app available in Japanese? If not, I think it's better to leave "Variables" in English